jobs/ci-kubernetes-e2e-gce-1-6-master-upgrade-cluster-new.env

```diff
@@ -0,0 +1,13 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/latest
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
+JENKINS_PUBLISHED_VERSION=ci/latest-1.6
+JENKINS_USE_SKEW_TESTS=true
+KUBE_NODE_OS_DISTRIBUTION=debian
+PROJECT=gce-up-c1-3-g1-4-up-clu-n
```

Comment:
Looks like we are recycling those projects, are we sure those projects are re-configured to match the current state?